src/sparql/QueryRepresentation.ts

```diff
@@ -7,5 +7,5 @@ export type Condition = {
 };
 
 export default interface QueryRepresentation {
-	condition: Condition;
+	conditions: Condition[];
```

Comment:
I think for the purpose of this PR it is ok if this stays a single condition for now. We'll extend that in a follow-up PR